scripts/validate_pr_title.py

```diff
@@ -0,0 +1,20 @@
+#-*- coding: utf-8-unix -*-
+import re
+import os
+import sys
+
+# Get the PR title from an environment variable
+pr_title = os.getenv("PR_TITLE", "").strip()
+
+# Define the conventional commit pattern
+pattern = r"^(feat|fix|docs|style|refactor|test|chore)(\(.+\))?: .+"
+
+# Validate the PR title
+if not re.match(pattern, pr_title):
```

Comment:
This function is near identical to the validate_commit_message.py - just a different input and error message. 
Why duplication again?